### PR TITLE
Improve and re-organize text

### PR DESCRIPTION
**Pull request by avpatel**
This PR only improves and re-organizes text without changing the functionality. Repetitive statements have also been removed at number of places.

### CODE REVIEW

**Comment by lftan** [comment]
Not sure why can't see my review comment above. Print screen

image

**Comment by lftan** [comment]
another :

image

File: src/service-groups.adoc [hunk 1]
```diff
@@ -43,7 +53,7 @@ The following table lists the service groups.
 | Service Group ID
 | Version (Major:Minor)
 | Service Group Name
-| RISC-V Privilege Mode
+| Allowed RISC-V Privilege Levels on APs
```
Comment (pathakraul) [inline src/service-groups.adoc:56]: %s/APs/application processors

Comment (pathakraul) [reply src/service-groups.adoc:56]: @lftan I am able to add the comment. You should only click the "Add single comment" instead of "Start the Review"

**Comment by avpatel** [comment]
> Not sure why can't see my review comment above. Print screen
> 
> image

We don't need to be specific about the scope of fast channels here. I will make the statement more generic.

File: src/service-groups.adoc (hunk 1 above)
Comment (avpatel) [reply src/service-groups.adoc:56]: @pathakraul Okay, I will update.

**Comment by avpatel** [comment]
> another :
> 
> image

These two requirements should actually be part of same requirement because a service group can be partially implemented only if it provides way to discover implemented services. This is very similar to how SBI extensions.

I will merge these two requirements so that readers see this as one requirement and don't read it out of context.

**Comment by avpatel** [comment]
@lftan @pathakraul I have addressed your comments. Let me know if you have more comments.

**Comment by pathakraul** [comment]
Overall, changes looks fine to me. Thanks @avpatel for improvement.

**Comment by lftan** [comment]
Look good. Merged

**Pull request merged**